src/node_quic_util.h

```diff
@@ -4,6 +4,7 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
 #include "node_quic_buffer.h"
+#include "util-inl.h"
```

Comment:
What is this one needed for? It’s probably a good sign that something should go into a `node_quic_util-inl.h` or `node_quic_util.cc`.

(Like, this isn’t important, it’s just something that should be figured out in the long run)

Reply:
It was required to fix this error:
```
In file included from ../src/node_quic_crypto.cc:1:
In file included from ../src/node_quic_crypto.h:6:
In file included from ../src/node_crypto.h:30:
In file included from ../src/env.h:27:
In file included from ../src/aliased_buffer.h:7:
../src/util.h:264:43: error: inline function 'node::ContainerOf<uv_timer_s, node::quic::Timer>' is not defined [-Werror,-Wundefined-inline]
constexpr ContainerOfHelper<Inner, Outer> ContainerOf(Inner Outer::*field,
                                          ^
../src/node_quic_util.h:439:22: note: used here
          Timer* t = ContainerOf(
```
I'd be happy to take a closer look at this tomorrow.

Reply:
Yeah, the `Timer` class methods shouldn’t be in this header :)

Reply:
Yep, I was going to move those at some point :-)